.buildkite/release.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
```

Comment:
What you can do here is to use the Makefile provided.

So instead of defining a sh script you can enrich the makefile to push the zip to S3 running `make release` (for example)